### PR TITLE
Price Prediction Model

### LINKED ISSUE #11: Pre-processing
- [ ] Normalization: Min-max implementation
- [ ] Removing the outlier using IQR

### LINKED ISSUE #12: Model Neural Network Training
- [ ] Split the training and testing dataset
- [ ] K-fold cross validation
- [ ] Hyperparameter

### PR DESCRIPTION
**Pull request by SamvathnaEm**

### CODE REVIEW

**Comment by Markcmd** [comment]
Amazing Job!
